src/core/buildFunctions.js

```diff
@@ -560,33 +560,25 @@ function getAverage( triangleBounds, offset, count, axis ) {
 function computeTriangleBounds( geo, fullBounds ) {
 
 	const posAttr = geo.attributes.position;
-	const posArr = posAttr.array;
 	const index = geo.index.array;
 	const triCount = index.length / 3;
 	const triangleBounds = new Float32Array( triCount * 6 );
-
-	// support for an interleaved position buffer
-	const bufferOffset = posAttr.offset || 0;
-	let stride = 3;
-	if ( posAttr.isInterleavedBufferAttribute ) {
-
-		stride = posAttr.data.stride;
-
-	}
+	const getter = [ posAttr.getX.bind( posAttr ), posAttr.getY.bind( posAttr ), posAttr.getZ.bind( posAttr ) ];
```

Comment:
This is probably the strangest line in this PR, and I don't like it very much; it was a way to be able to use the correct getter in the upcoming triangle for loop, since the loop examines the x values for all the vertices in the triangle, then the y, then the z. And I didn't really want to write a three-way switch statement to do that, or really dig too deep into the logic.

There's probably a better way to handle this, even if something as simple as a better name than `getter`.

Reply:
I will propose a new function in three.js that will make this easier and we'll see where that goes.

Reply:
For now I've proposed https://github.com/mrdoob/three.js/pull/24515 to support this type of use case more easily. I'm hoping something along these lines in the next three.js release but in the mean time why don't we use an array of strings to index the function names as I'd done here:

https://github.com/mrdoob/three.js/blob/dev/examples/jsm/utils/BufferGeometryUtils.js#L410-L435